crates/hyperqueue/src/common/utils/time.rs

```diff
@@ -67,7 +67,7 @@ pub mod mock_time {
     use std::time::Instant;
 
     thread_local! {
-        static MOCK_TIME: RefCell<Option<Instant>> = RefCell::new(None);
+        static MOCK_TIME: RefCell<Option<Instant>> = const { RefCell::new(None) };
```

Comment:
Isn't `inline const` only stabilized in Rust `1.79`?

Reply:
It's fine to upgrade, but it hasn't been released yet, that's why I was surprised :laughing: I realized now that this is actually some built-in functionality of the `thread_local!` macro (https://doc.rust-lang.org/std/macro.thread_local.html).